README.md

```diff
@@ -1,6 +1,5 @@
 <div align="center">
 
-[![Test Coverage](https://api.codeclimate.com/v1/badges/407cdb68165fc000cfd6/test_coverage)](https://codeclimate.com/github/CatalysmsServerManager/7-days-to-die-server-manager/test_coverage)
```

Comment:
Can we replace this with the badge for codecov?

```
[![codecov](https://codecov.io/gh/CatalysmsServerManager/7-days-to-die-server-manager/branch/master/graph/badge.svg)](https://codecov.io/gh/CatalysmsServerManager/7-days-to-die-server-manager)

```

Reply:
Yes we can do it....
let me add that one

Reply:
Could you review my changes?